static/app/views/insights/common/components/modulePageFilterBar.tsx

```diff
@@ -70,21 +64,24 @@ export function ModulePageFilterBar({moduleName, onProjectChange, extraFilters}:
 
   const dateFilterProps: DatePageFilterProps = {};
   if (hasDateRangeQueryLimit) {
-    dateFilterProps.relativeOptions = {
-      '1h': t('Last 1 hour'),
-      '24h': t('Last 24 hours'),
-      '7d': t('Last 7 days'),
-      '14d': <DisabledDateOption value={t('Last 14 days')} />,
-      '30d': <DisabledDateOption value={t('Last 30 days')} />,
-      '90d': <DisabledDateOption value={t('Last 90 days')} />,
+    dateFilterProps.relativeOptions = ({arbitraryOptions}) => {
+      return {
+        ...arbitraryOptions,
+        '1h': t('Last 1 hour'),
+        '24h': t('Last 24 hours'),
+        '7d': t('Last 7 days'),
+        '14d': <DisabledDateOption value={t('Last 14 days')} />,
+        '30d': <DisabledDateOption value={t('Last 30 days')} />,
+        '90d': <DisabledDateOption value={t('Last 90 days')} />,
+      };
     };
+
     dateFilterProps.maxPickableDays = QUERY_DATE_RANGE_LIMIT;
     dateFilterProps.isOptionDisabled = ({value}) => {
-      if (value === 'absolute') {
+      if (!['14d', '30d', '90d'].includes(value)) {
```

Comment:
minor suggestion, we could store these in a constant somewhere above

Reply:
Done!